packages/flame_fire_atlas/lib/flame_fire_atlas.dart

```diff
@@ -428,4 +428,9 @@ class FireAtlas {
       loop: selection.loop,
     );
   }
+
+  /// Returns the atlas image.
+  ///
+  /// Throws if called before the image is loaded.
+  Image get atlasImage => _assertImageLoaded();
```

Comment:
I agree with the `_assert` name, just didn't wanted to refactor, there are many things that I want to eventually refactor on Fire atlas.

About the `image` thing, I thought on that too, I just worried that it could be a bit too generic and atlasImage would be more explicit? But I am happy to change to image if we feel that it is enough!

Reply:
> About the image thing, I thought on that too, I just worried that it could be a bit too generic and atlasImage would be more explicit? But I am happy to change to image if we feel that it is enough!

Since it is on the `FireAtlas` object it feels like it should be clear enough that the image is for the atlas?

Reply:
Fair enough!

```suggestion
  Image get image => _assertImageLoaded();
```